main/fakeroot/APKBUILD

```diff
@@ -4,30 +4,31 @@ pkgver=1.23
 pkgrel=0
 pkgdesc="Gives a fake root environment, useful for building packages as a non-privileged user"
 arch="all"
-license='GPL'
-url="http://fakeroot.alioth.debian.org/"
+license="GPL-3.0-or-later"
+url="https://packages.debian.org/fakeroot"
 depends=
 options="!checkroot"
 checkdepends="bash"
 makedepends_build="libtool autoconf automake po4a"
 makedepends_host="libcap-dev acl-dev linux-headers"
 makedepends="$makedepends_build $makedepends_host"
 subpackages="$pkgname-doc"
-source="http://ftp.debian.org/debian/pool/main/f/$pkgname/${pkgname}_${pkgver}.orig.tar.xz
+source="http://ftp.debian.org/debian/pool/main/f/fakeroot/${pkgname}_${pkgver}.orig.tar.xz
```

Comment:
IMHO, this is a useful change, which makes it easier to copy paste the url to look for new versions. :+1: